### ISSUE TITLE
installer needs to allow users to de-select virtual

### ISSUE THREAD

**Issue by SvenDowideit** [docker/toolbox#349]
When the user has Hyper-V installed and in use, then installing VBox is way too risky - and (I'm presuming) that if I create my own docker-machine hyper-v machine called default, at least some of toolbox will work?

(I'm essentially looking to install docker-client, docker-machine and have the docker-cli work - kitematic working is a bonus.

I clicked "dont install drivers from oracle" we'll see how bad things get :)

**Comment by SvenDowideit** [comment]
it may look ok in theory, but I'm doing this on a win2016 server tp4 installed on an older notebook, so hyper-v is different. maybe later on the other box.

**Comment by jeffdm** [comment]
Thanks Sven!

**Comment by SvenDowideit** [comment]
I threw the SSD into my modern desktop box, and now have a docker-machine hyperv b2d, I'll raise more issues/prs if i have time to figure out how to make toolbox use it :)

**Comment by zh99998** [comment]
I'm using windows server 2012, it works

just need manually create a default machine and commend out 

```bash
if [ ! -f $DOCKER_MACHINE ] || [ ! -f "${VBOXMANAGE}" ]; then
  echo "Either VirtualBox or Docker Machine are not installed. Please re-run the Toolbox Installer and try again."
  exit 1
fi
```

in start.sh

**Comment by lordvlad** [comment]
+1
I have hyper-v installed and cannot deselect 'virtualbox' during the docker toolbox installation.

**Comment by Nox-404** [comment]
I agree, the installer should not force the installation of virtualbox

**Comment by pmario** [comment]
Any progress on this issue?  The [readme.md FAQ](https://github.com/docker/toolbox/blob/master/README.md#frequently-asked-questions) states:

>Do I have to install VirtualBox?

>No, you can deselect VirtualBox during installation. It is bundled in case you want to have a working environment for free.

Which is a bit confusing!

**Comment by archergod** [comment]
I have Hyper-V installed and I need that for some of my .NET stuff/Visual Studio. In installer for Windows I didn't see an option to disable virtualbox, Kinematic is not running as well as it need Hyper-V been disabled.

**Comment by pmario** [comment]
Kitematic doesn't support hyper-v at the moment. .. there is some work under way, but help will be much appreciated.  see: #1463

**Comment by pmario** [comment]
@jeffdm ... What does it need to fix this annoying and long standing issue? ... The Toolbox installer is useless if experienced users are not allowed to deselect elements if the "Custom Installation" is chosen.  

Full Install option:

![toolbox-1](https://cloud.githubusercontent.com/assets/374655/14526485/d64a1c0e-0243-11e6-9335-82d420abb713.gif)

Custom Install option:

![toolbox-2](https://cloud.githubusercontent.com/assets/374655/14526498/e3218dd6-0243-11e6-98be-0ac27a737122.gif)

So the installer is capable of handling the issue. IMO it just needs to be enabled!
